src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```diff
@@ -115,6 +115,9 @@ main.extended.help.footer=\
 error.prefix=Error:
 warn.prefix=Warning:
 
+err.runtime.link.not.linkable.runtime=The current run-time image does not support run-time linking.
+err.runtime.link.jdk.jlink.prohibited=Including jdk.jlink module for run-time image based links is not allowed.
```

Comment:
The phrase "run-time image based links" in this error message (and in the value of err.runtime.link.packaged.mods) is a bit unusual. As developers will see this message then maybe it could say that this jlink in the current run-time image doesn't contain packaged modules and cannot be used to create another run-time image that include the jdk.jlink module.

Reply:
I've used alternative phrasing. Hopefully better now.